cli-tests/package.json

```diff
@@ -7,7 +7,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "test": "npx jest --verbose --testPathPattern=common.test.ts"
+    "test": "npx jest --verbose --testPathPattern="
```

Comment:
Maybe, the whole `--testPathPattern=` could be removed in this case?

Reply:
We definitely could remove it. I keep it on purpose to simplify test run using a specific test file. So to do this you just have to copy this string or insert any file test that you want to run

Reply:
Was discussed in DM to keep it as is.